index.html

```diff
@@ -104,9 +108,216 @@ <h2 class="color-heading" id="heading-2">
+                <li>HTML</li>
+              </ul>
+            </li>
+            <li class="display-flex-width btn-project-1">
+              <button class="action-button" type="button">See Project</button>
+            </li>
+          </ul>
+        </div>
+
+        <div class="section-container2 section-box item-grid">
+          <ul class="first-work display-flex-2">
+            <li class="project1-image display-flex-width">
+              <img
+                class="p1"
+                src="assets/images/work-project2.png"
+                alt="Multi Post Stories: Personalized Stories selection"
+                rel="noopener noreferrer"
+              />
+            </li>
+            <li class="display-flex-width heading-height">
+              <h3 class="project-heading">Multi Post Stories</h3>
+            </li>
+            <li class="display-flex-width languages-used">
+              <ul class="languages language-flex">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+            </li>
+            <li class="display-flex-width btn-project-1">
+              <button class="action-button" type="button">See Project</button>
+            </li>
+          </ul>
+        </div>
+
+        <div class="section-container3 section-box item-grid">
+          <ul class="first-work display-flex-3">
+            <li class="project1-image display-flex-width">
+              <img
+                class="p1"
+                src="assets/images/work-project3.png"
+                alt="Health and Fitness Project: Personalized Stories selection"
+                rel="noopener noreferrer"
+              />
+            </li>
+            <li class="display-flex-width heading-height">
+              <h3 class="project-heading">Health and Fitness Project</h3>
+            </li>
+            <li class="display-flex-width languages-used">
+              <ul class="languages language-flex">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+            </li>
+            <li class="display-flex-width btn-project-1">
+              <button class="action-button" type="button">See Project</button>
+            </li>
+          </ul>
+        </div>
+        <div class="section-container2 section-box item-grid">
+          <ul class="first-work display-flex-4">
+            <li class="project1-image display-flex-width">
+              <img
+                class="p1"
+                src="assets/images/work-project4.png"
+                alt="Art Prinitngs: Personalized Art Selection"
+                rel="noopener noreferrer"
+              />
+            </li>
+            <li class="display-flex-width heading-height">
+              <h3 class="project-heading">Art Printing Portfolio</h3>
+            </li>
```

Comment:
- [ ] Use `a` a tag that will redirect to the project specifications like live demo link or on GitHub.

Reply:
I don't have these projects actually set up, this is just me following a template. I can connect these links to my personal github profile.